src/elements/EnvironmentVariables/EnvironmentVariables.js

```diff
@@ -0,0 +1,165 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+
+import EnvironmentVariablesView from './EnvironmentVariablesView'
+
+import {
+  newVariableInitialState,
+  SECRET_TYPE,
+  validationInitialState,
+  VALUE_TYPE
+} from './environmentVariables.util'
+
+import { ReactComponent as Edit } from '../../images/edit.svg'
+import { ReactComponent as Delete } from '../../images/delete.svg'
+
+const EnvironmentVariables = ({
+  className,
+  handleAddNewEnv,
+  handleDeleteEnv,
+  handleEditEnv,
+  variables
+}) => {
+  const [newVariable, setNewVariable] = useState(newVariableInitialState)
+  const [validation, setValidation] = useState(validationInitialState)
+  const [showAddNewVariableRow, setShowAddNewVariableRow] = useState(false)
+  const [selectedVariable, setSelectedVariable] = useState(null)
```

Comment:
```suggestion
  const [selectedEnvVariable, setSelectedEnvVariable] = useState(null)
```

Please rename everywhere variable to envVariable